src/tests/running.md

```diff
@@ -19,6 +19,10 @@ timestamp file for every test can be found under `build/ARCH/test/`.
 To force-rerun a test (e.g. in case the test runner fails to notice
 a change) you can simply remove the timestamp file.
 
+Note that some tests require a Python-enabled gdb. If you are building
+gdb from source, you need to configure with
+`--with-python=<path-to-python-binary>`.
```

Comment:
Perhaps it might be good to mention how to check if your gdb is python-enabled?

Reply:
> type some Python code

Not all of us know what valid Python code looks like.

Reply:
Perhaps give an example...

"type some python code (e.g. `print('hello gdb')`)"